docs/zh/samples/dataset_across_namespace.md

```diff
@@ -0,0 +1,151 @@
+# 示例 - Dataset缓存跨Namespace访问
+本示例用来演示如何一份Dataset缓存数据，如何跨Namespace使用：
+- Namespace ns-a 创建 Dataset demo 和 AlluxioRuntime demo
+- Namespace ns-b 创建 Dataset demo-ref 和 ThinRuntime demo-ref，其中demo-ref  mount的路径为`dataset://ns-a/demo"
+ 
+## 前提条件
+在运行该示例之前，请参考[安装文档](../userguide/install.md)完成安装，并检查Fluid各组件正常运行：
+```shell
+$ kubectl get pod -n fluid-system
+alluxioruntime-controller-5b64fdbbb-84pc6   1/1     Running   0          8h
+csi-nodeplugin-fluid-fwgjh                  2/2     Running   0          8h
+csi-nodeplugin-fluid-ll8bq                  2/2     Running   0          8h
+dataset-controller-5b7848dbbb-n44dj         1/1     Running   0          8h
+thinruntime-controller-7dcbf5f45-xsf4p          1/1     Running   0          8h
+```
+
+其中，`thinruntime-controller`是用来支持Dataset跨Namespace共享，`alluxioruntime-controller`是实际的缓存。
+
+## CSI机制跨Namespace共享数据集缓存
+###  1. 创建Dataset和缓存Runtime
+
+在默认名空间下，创建`phy` Dataset和AlluxioRuntime
+```shell
+$ cat<<EOF >ds.yaml
+apiVersion: data.fluid.io/v1alpha1
+kind: Dataset
+metadata:
+  name: phy
+spec:
+  mounts:
+    - mountPoint: https://mirrors.bit.edu.cn/apache/spark/
+      name: spark
+---
+apiVersion: data.fluid.io/v1alpha1
+kind: AlluxioRuntime
+metadata:
+  name: phy
+spec:
+  replicas: 1
+  tieredstore:
+    levels:
+      - mediumtype: MEM
+        path: /dev/shm
+        quota: 1Gi
+        high: "0.95"
+        low: "0.7"
+EOF
+
+$ kubectl create -f ds.yaml
+```
+
+### 2. 创建引用的Dataset和Runtime
+在 ref 名空间下，创建：
+- 引用的数据集`refdemo`，其mountPoint格式为`dataset://${origin-dataset-namespace}/${origin-dataset-name}`；
+- ThinRuntime `refdemo`，其 profileName 必须为空值；
+注：当前引用的数据集，只支持一个mount，且形式必须为`dataset://`（即出现`dataset://`和其它形式时，dataset创建失败）。
+```shell
+$ kubectl create ns ref
+
+$ cat<<EOF >ds-ref.yaml
+apiVersion: data.fluid.io/v1alpha1
+kind: Dataset
+metadata:
+  name: refdemo
+spec:
+  mounts:
+    - mountPoint: dataset://default/phy
+      name: fusedemo
+---
+apiVersion: data.fluid.io/v1alpha1
+kind: ThinRuntime
+metadata:
+  name: refdemo
+spec:
+  profileName: ""
```

Comment:
Do we have to set `profileName` ? Or just 

```yaml
apiVersion: data.fluid.io/v1alpha1
kind: ThinRuntime
metadata:
  name: refdemo
```

Reply:
fixed, no need to set.